code/game/gamemodes/miniantags/abduction/gland.dm

```diff
@@ -75,7 +75,8 @@
 	owner << "<span class='warning'>You feel nauseous!</span>"
 	owner.vomit(20)
 
-	var/mob/living/simple_animal/slime/Slime = new/mob/living/simple_animal/slime(get_turf(owner))
+	var/mob/living/simple_animal/slime/Slime
+	Slime = new(get_turf(owner), "grey")
```

Comment:
@coiax develops in a small 80 character terminal, they've been told they can space STRINGS along multiple lines but has been told off before for doing it to CODE lines.